### PR TITLE
Fix build and fix dowload kubeconfig

### PR DESCRIPTION
**Pull request by m-ildefons**
Fix instructions for downloading kubeconfig file for Rancher Terraform instructions:
- Fix the syntax highlighting
- Add instructions for downloading the Kubeconfig file via web UI in line with previous instructions

Fix build: fix inconsistent entries in the versioned docs v1.3 sidebar.js

### CODE REVIEW

File: docs/rancher/rancher-terraform.md [hunk 1]
```diff
@@ -39,6 +39,17 @@ The [Rancher Terraform Provider](https://registry.terraform.io/providers/rancher
 
 1. Obtain the kubeconfig for the Harvester Cloud Provider and the Harvester CSI Driver.
 
+    <Tabs>
+    <TabItem value="ui" label="UI" default>
+
+    On the Rancher UI, go to **Virtualization Management** and select
+    **Download Kubeconfig** from the context menu of the corresponding Harvester
+    Cluster.
```
Comment (jillian-maroket) [inline docs/rancher/rancher-terraform.md:47]:
```suggestion
    On the Rancher UI, go to **Virtualization Management**. Locate the target Harvester cluster in the list and then select **⋮** > **Download KubeConfig**.
```

Comment (jillian-maroket) [reply docs/rancher/rancher-terraform.md:47]: For most users, a series of menu choices is easier to understand than UI-specific terms such as "context menu".

**Review by jillian-maroket — COMMENTED** [review]
Thank you for fixing the indentation issue. I suggested a small wording change.

File: docs/rancher/rancher-terraform.md (hunk 1 above)
Comment (m-ildefons) [reply docs/rancher/rancher-terraform.md]: Good point. Thanks :smile:

**Review by jillian-maroket — APPROVED** [review]

**Pull request merged**